src/test/java/de/otto/teamdojo/web/rest/SkillResourceIntTest.java

```diff
@@ -56,8 +56,12 @@
     private static final String DEFAULT_VALIDATION = "AAAAAAAAAA";
     private static final String UPDATED_VALIDATION = "BBBBBBBBBB";
 
-    private static final String DEFAULT_EXPIRY_PERIOD = "P+5M+24D";
-    private static final String UPDATED_EXPIRY_PERIOD = "P78Y3W";
+    //    private static final String DEFAULT_EXPIRY_PERIOD = "P+5M+24D";
```

Comment:
Looks like a leftover. Could be deleted.

Reply:
Removed.